README.md

```diff
@@ -1,20 +1,14 @@
-# Announcement Center
+# Announcement center
 
-This app allows admins to post announcements!
+![Frontpage of the announcement center app with teh admin form](docs/AnnouncementCenterFrontpage.png)
```

Comment:
```suggestion
![](docs/AnnouncementCenterFrontpage.png)
```
Typo with the "teh" in the alt text – but since the screenshot is decorative, no alt text is needed anyway. :)